### ISSUE TITLE
Alternate simulated touchpad input mode

### ISSUE THREAD

**Issue by lightrush** [iiordanov/remote-desktop-clients#147]
- Everything the same as the current Simulated touchpad mode
- Press and hold does nothing
- Double-tap and drag (tap once, quickly tap a second time and drag without releasing after second tap) initiates drag. Lifting after that initiates drop

**Comment by bpsib** [comment]
I fully agree. I find also the vibration annoying when tap and holding. There's also occasions where it's over-sensitive where it keeps thinking I am holding it unless I use it swiftly. I think it's best for it to function just like a laptop touchpad as described in the post above. I can also get things that involve lots of drag and dropping done quicker that way.

**Comment by iiordanov** [comment]
Vibration on tap issue was fixed.

Double-tap drag already works in all input modes, even though press-hold does something.
